includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -1539,15 +1538,17 @@ public static function enqueue_edit_post_screen_scripts() {
 	 */
 	public static function add_edit_post_inline_script() {
 		$current_screen = get_current_screen();
-		if ( 'post' !== $current_screen->base || self::POST_TYPE_SLUG !== $current_screen->post_type ) {
+		if ( 'post' !== $current_screen->base || self::POST_TYPE_SLUG !== $current_screen->post_type || ! isset( $_GET['post'] ) ) { // WPCS: CSRF OK.
 			return;
 		}
 
+		$post = get_post( intval( $_GET['post'] ) );  // WPCS: CSRF OK.
```

Comment:
I don't think this is necessary because the `$post` global will already be populated on the edit post screen. So instead of `$post` this you can just use `get_post()` below.